content/en/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -32,11 +32,8 @@ DaemonSet has two update strategy types:
 To enable the rolling update feature of a DaemonSet, you must set its
 `.spec.updateStrategy.type` to `RollingUpdate`.
 
-You may want to set
-[`.spec.updateStrategy.rollingUpdate.maxUnavailable`](/docs/concepts/workloads/controllers/deployment/#max-unavailable)
-(default to 1) and
-[`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds)
-(default to 0) as well.
+You may want to set [`.spec.updateStrategy.rollingUpdate.maxUnavailable`](/docs/concepts/workloads/controllers/deployment/#max-unavailable) (default
+to 1), [`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) (default to 0) [`.spec.maxSurge`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-surge) (a beta feature and defaults to 25%) as well.
```

Comment:
The current line wrapping has an implicit style, i.e. wrap long lines when possible except for hyper links.
Please consider do the same so that 1) future change tracking is easier; 2) localization teams can easily
spot where they fall out of sync.

Reply:
So, should I have a single line?

Reply:
Hi @ravisantoshgudimetla ,
The updated content should wrap the long lines, except for the hyperlinks.
You can follow the example set by the original content?

Reply:
Thank you @tengqm and @kbhawkey for clarification. I updated the lines as per your suggestions. PTAL